docs/src/modules/components/AppLayoutDocs.js

```diff
@@ -37,20 +37,30 @@ const Main = styled('main', {
 }));
 
 const StyledAppContainer = styled(AppContainer, {
-  shouldForwardProp: (prop) => prop !== 'disableAd',
-})(({ disableAd, theme }) => {
+  shouldForwardProp: (prop) => prop !== 'disableAd' && prop !== 'hasTabs',
+})(({ disableAd, hasTabs, theme }) => {
   return {
     position: 'relative',
     // By default, a grid item cannot be smaller than the size of its content.
     // https://stackoverflow.com/questions/43311943/prevent-content-from-expanding-grid-items
     minWidth: 0,
     ...(!disableAd && {
-      '&& .description': {
-        marginBottom: 198,
-      },
-      '&& .description.ad': {
-        marginBottom: 40,
-      },
+      ...(!hasTabs && {
+        '&& .description': {
+          marginBottom: 198,
+        },
+        '&& .description.ad': {
+          marginBottom: 40,
+        },
+      }),
+      ...(hasTabs && {
+        '&& .component-tabs .MuiTabs-root': {
+          marginBottom: 193,
+        },
+        '&& .component-tabs.ad .MuiTabs-root': {
+          marginBottom: 35,
+        },
+      }),
```

Comment:
There is a layout shift when the ad loads:

<img width="769" alt="Frame 1 (1)" src="https://user-images.githubusercontent.com/3165635/229188573-d5e45c53-917c-41e0-9036-5fb796fc7d4f.png">